.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Mr Julian -> please add a .gitignore file on the main level of the project and add the .idea + target folders to it :D 
They are autogenerated and there is no use to keep that in the repo -> only makes it harder to check out the code and make changes